templates/global/global.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+<link rel="stylesheet" type="text/css" href="{% static 'css/global.css' %}" />
+<body>
+  <div class="sidebar-container">{% include 'global/sidebar.html' %}</div>
+  <div class="table-container">{% include 'global/table.html' %}</div>
```

Comment:
You make Django code to look like React code, that is very good!

Reply:
Thats how i think so thats how i code :) 
thanks man